app/assets/javascripts/admin/addon/templates/modal/admin-show-post-edits-user.hbs

```diff
@@ -0,0 +1,21 @@
+{{#d-modal-body class="post-edits-modal" title="admin.user.post_edits"}}
+
+  {{#conditional-loading-spinner condition=loadingUser}}
+
+    {{admin-report
+            showFilteringUI=false
+            showAllReportsLink=false
+            dataSourceName=model.type
+            filters=model
+            showTitle=false
+            reportOptions=null
+    }}
+
+  {{/conditional-loading-spinner}}
+
+{{/d-modal-body}}
+
+<div class="modal-footer">
+  {{d-modal-cancel close=(route-action "closeModal")}}
+  {{conditional-loading-spinner condition=loading size="small"}}
```

Comment:
Out of curiosity, what is the loading spinner here used for?

Reply:
Since this is my first time using Ember on a production project, this is an artifact of copy pasting, I'll remove it.